src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -103,6 +113,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
+    void SettingContainer::_OnCurrentValueChanged(const Windows::UI::Xaml::DependencyObject& d, const Windows::UI::Xaml::DependencyPropertyChangedEventArgs& /*e*/)
+    {
+        const auto& obj{ d.try_as<Editor::SettingContainer>() };

```

Comment:
```suggestion
        const auto obj{ d.try_as<Editor::SettingContainer>() };
```

not necessary; decays to `const auto` anyway